### PR TITLE
Hides update description if the update is marked as backers only

### PR DESCRIPTION
**Pull request by zcoursey22**

### CODE REVIEW

File: client/src/components/update.jsx [hunk 1]
```diff
@@ -4,15 +4,24 @@ import helpers from '../../../helpers/helpers.js';
 const Update = (props) => {
   const date = props.update.date;
   const formattedDate = `${helpers.monthNumberToString(date.getMonth())} ${date.getDate()}, ${date.getFullYear()}`;
+
+  let desc = null;
+  let backersOnly = null;
+  if (!props.update.backersOnly) {
+    desc = <p>{props.update.description}</p>
```
Comment (despinoz) [inline client/src/components/update.jsx]: I believe the tags and the content should be on different lines

File: client/src/components/update.jsx [hunk 2]
```diff
@@ -4,15 +4,24 @@ import helpers from '../../../helpers/helpers.js';
...
+  } else {
+    backersOnly = <span>For backers only</span>
```
Comment (despinoz) [inline client/src/components/update.jsx]: Same here

**Review by despinoz — CHANGES_REQUESTED** [review]
@zach do you have ESLint enabled? I left you specific comments on the code. Also consider to use 'moment js' for the date format. It can be another ticket if you want.

**Comment by zcoursey22** [comment]
I'm going to be refactoring with moment.js later on

**Review by despinoz — APPROVED** [review]

**Comment by despinoz** [comment]
As we discussed, prioritize the ticket with the ESLint configuration. Also resolve the conflict showed on Github before merge

**Pull request merged**